debug/CMakeLists.txt

```diff
@@ -7,13 +7,9 @@ set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE CACHE STRING "" FORCE)
 set(CPACK_RPM_PACKAGE_LICENSE "MPLv2.0")
 set(CPACK_STRIP_FILES TRUE)
 
-remove_definitions(-DSTS_COMPILE_UNIT_TESTS)
+include_directories(${CMAKE_CURRENT_SOURCE_DIR})
 
-set(DEBUG_LIBS
-    ${CMAKE_CURRENT_SOURCE_DIR}/debug.h    
-)
-
-add_executable(sax_convert $ENV{SYMTSERIES_SRC} ${DEBUG_LIBS} sax_convert.c)
-add_executable(fetch_data $ENV{SYMTSERIES_SRC} ${DEBUG_LIBS} fetch_data.c)
-
-install(TARGETS sax_convert DESTINATION bin)
+add_executable(sax_convert sax_convert.c)
+target_link_libraries(sax_convert symtseries m)
```

Comment:
Use find_library (http://www.cmake.org/cmake/help/v3.0/command/find_library.html) instead of just blindly including it.